src/views/email-2022.js

```diff
@@ -32,14 +32,7 @@ const tableStyle = `
   width: 100%;
 `
 
-const headeLogoStyle = `
-  background-size: 240px 50px;
-  background: #f9f9fa url('${images.logoLight}')
-  height: 100px;
-  no-repeat 50%;
-`
```

Comment:
This style is broken and should be:
```
const headeLogoStyle = `
  background-size: 240px 50px;
  background: #f9f9fa url('${images.logoLight}') no-repeat 50%;
  height: 100px;
`
```

But even then the inline style got stripped and the issue remained.